prometheusbp/promtest/testutil.go

```diff
@@ -40,6 +40,50 @@ func (p *PrometheusMetricTest) CheckSampleCountDelta(delta int) {
 	}
 }
 
+// CheckDeltaLE checks that the metric value changes less or equal to delta from when Helper was called.
+func (p *PrometheusMetricTest) CheckDeltaLE(delta float64) {
```

Comment:
I'm adding Prometheus metric tests for [auction/inference.go UpdateWithInferenceResponse](https://github.snooguts.net/reddit/ads-go/blob/master/ad_selector/auction/inference.go#L151) and there's some actions in [inference_test.go/TestInferPCE](https://github.snooguts.net/reddit/ads-go/blob/master/ad_selector/auction/inference_test.go#L111) and TestInferPCEFailedBatches because of [this part](https://github.snooguts.net/reddit/ads-go/blob/master/ad_selector/auction/inference.go#L151).